lib/exercism_config/generate_aws_settings.rb

```diff
@@ -0,0 +1,42 @@
+module ExercismConfig
+  class GenerateAwsSettings
+    include Mandate
+
+    def call
+      {
+        region: 'eu-west-2',
+        profile: profile,
+        access_key_id: aws_access_key_id,
+        secret_access_key: aws_secret_access_key
+      }.select {|k,v|v}
+    end
+
+    memoize
+    def aws_access_key_id
+      case Exercism.environment
+      when :development, :test
+        "FAKE"
+      else 
+        nil
+      end
+    end
+
+    memoize
+    def aws_secret_access_key
+      case Exercism.environment
+      when :development, :test
+        "FAKE"
+      else 
+        nil
```

Comment:
Not needed to explicitly set it to `nil`, however I understand why it might be done anyway.

Reply:
Yeah, I always like to be explicit about `nil`. But thank you for the pointer.